examples/render-helm-chart-crds/.expected/config.yaml

```diff
@@ -0,0 +1,19 @@
+# This test should be skipped because a bug in the kpt orchestrator
+# results in non-idempotent behavior.
+
+# Upon the first run, the contents of customresourcedefinition_workspaces.app.terraform.io.yaml
+# contains the following:
+
+#  creationTimestamp: null
+#  name: workspaces.app.terraform.io
+
+# On the second run, however, these fields are reversed:
+
+#  name: workspaces.app.terraform.io
+#  creationTimestamp: null
+
+# kpt should preserve the ordering of fields. Ordering is preserved if creationTimestamp
+# is anything except null (correct behavior), but is not preserved when creationTimestamp
+# is null.
+
+skip: true
```

Comment:
I guess we don't have to skip it, right?
Even there may be an issue with the fields ordering, we can tweak the fields ordering in the input to reduce the unnecessary diff.

Reply:
We can't tweak the fields in the input because the input is a remote file. One option I can try is to copy the charts locally and tweak the input.